File: talker_listener_int/src/talker_int.cpp
#include "ros/ros.h"
#include "std_msgs/Int32.h"

int main(int argc, char **argv)
{
 ros::init(argc,argv, "talker_int");
 ros::NodeHandle n;
 
 ros::Publisher int_chat_topic = n.advertise<std_msgs::Int32>("int_chatter", 1000);
 
 ros::Rate loop_rate(10);
 
 while (ros::ok())
 {
  std_msgs::Int32 msg;
  msg.data = 10;
  
  ROS_INFO("%d", msg.data);
  int_chat_topic.publish(msg);
  ros::spinOnce();
  loop_rate.sleep();
 }
 
 return 0;
}
  

File: talker_listener_int/scripts/talker_int.py
#!/usr/bin/env python3
import rospy
from std_msgs.msg import Int32

def talker_int():
    rospy.init_node('talker_int_py', anonymous=True)
    Pub_int = rospy.Publisher('int_chatter', Int32, queue_size=100)
    rate = rospy.Rate(10)


    while not rospy.is_shutdown():
        data = 19
        rospy.loginfo(data)
        Pub_int.publish(data)
        rate.sleep()

if __name__ == '__main__':
    try:
        talker_int()
    except rospy.ROSInterruptException:
        pass